Auto update failing for rtorrent

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [rtorrent](https://github.com/termux/termux-packages/blob/1cc3237342df086b296c6357b5915bc837544f1a/packages/rtorrent) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating rtorrent [Current version: 0.9.8]
INFO: package being updated to 0.10.0.
Downloading https://github.com/rakshasa/rtorrent/releases/download/v0.10.0/rtorrent-0.10.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0  393k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  393k  100  393k    0     0  1975k      0 --:--:-- --:--:-- --:--:-- 17.4M

You are about to commit these changes:

--------------------
diff --git a/packages/rtorrent/build.sh b/packages/rtorrent/build.sh
index 6d65c3777..af21ba9a7 100644
--- a/packages/rtorrent/build.sh
+++ b/packages/rtorrent/build.sh
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://rakshasa.github.io/rtorrent/
 TERMUX_PKG_DESCRIPTION="Ncurses BitTorrent client based on libTorrent"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.9.8
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_VERSION="0.10.0"
 TERMUX_PKG_SRCURL=https://github.com/rakshasa/rtorrent/releases/download/v${TERMUX_PKG_VERSION}/rtorrent-$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=9edf0304bf142215d3bc85a0771446b6a72d0ad8218efbe184b41e4c9c7542af
+TERMUX_PKG_SHA256=cc65bba7abead24151f10af116eca2342b0c320fdff3cb8d604c0af09215d3aa
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libc++, libcurl, libtorrent, libxmlrpc, ncurses"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
--------------------

bump(main/rtorrent): 0.10.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package rtorrent exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building rtorrent for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  78932      0 --:--:-- --:--:-- --:--:-- 78646
100 13134  100 13134    0     0  78883      0 --:--:-- --:--:-- --:--:-- 78646
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   5191      0 --:--:-- --:--:-- --:--:--  5206
gpg: Signature made Sun 29 Sep 2024 06:28:14 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  81113      0 --:--:-- --:--:-- --:--:-- 81219
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   5449      0 --:--:-- --:--:-- --:--:--  5480
gpg: Signature made Tue 24 Sep 2024 06:23:44 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  83701      0 --:--:-- --:--:-- --:--:-- 84102
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   5486      0 --:--:-- --:--:-- --:--:--  5516
gpg: Signature made Sat 28 Sep 2024 02:33:39 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@27b if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp2@1.63.0 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnghttp2/libnghttp2_1.63.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 99048  100 99048    0     0  1836k      0 --:--:-- --:--:-- --:--:-- 1860k
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp3@1.5.0 if necessary...
Found libnghttp3 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnghttp3/libnghttp3_1.5.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 63400  100 63400    0     0  1145k      0 --:--:-- --:--:-- --:--:-- 1168k
extracting libnghttp3 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.07.02 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.07.02 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.3.2 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.3.2 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libssh2@1.11.0 if necessary...
Found libssh2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libssh2/libssh2_1.11.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  193k  100  193k    0     0  2989k      0 --:--:-- --:--:-- --:--:-- 3018k
extracting libssh2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcurl@8.10.1 if necessary...
Found libcurl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libcurl/libcurl_8.10.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  884k  100  884k    0     0  12.3M      0 --:--:-- --:--:-- --:--:-- 12.5M
extracting libcurl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency curl@8.10.1 if necessary...
Found curl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/curl/curl_8.10.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  207k  100  207k    0     0  3085k      0 --:--:-- --:--:-- --:--:-- 3092k
extracting curl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libtorrent@0.13.8-4 if necessary...
Found libtorrent in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libt/libtorrent/libtorrent_0.13.8-4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  294k  100  294k    0     0   396k      0 --:--:-- --:--:-- --:--:--  396k
extracting libtorrent to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxmlrpc@1.59.03 if necessary...
Found libxmlrpc in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxmlrpc/libxmlrpc_1.59.03_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 11  270k   11 30615    0     0  54530      0  0:00:05 --:--:--  0:00:05 54475
100  270k  100  270k    0     0   353k      0 --:--:-- --:--:-- --:--:--  353k
extracting libxmlrpc to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses/ncurses_6.5.20240831_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  524k  100  524k    0     0  8454k      0 --:--:-- --:--:-- --:--:-- 8599k
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-utils/ncurses-utils_6.5.20240831_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 87768  100 87768    0     0  1597k      0 --:--:-- --:--:-- --:--:-- 1617k
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs/ncurses-ui-libs_6.5.20240831_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 33072  100 33072    0     0   581k      0 --:--:-- --:--:-- --:--:--  576k
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs-static/ncurses-ui-libs-static_6.5.20240831_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 39580  100 39580    0     0   807k      0 --:--:-- --:--:-- --:--:--  822k
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/rakshasa/rtorrent/releases/download/v0.10.0/rtorrent-0.10.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  393k  100  393k    0     0  2331k      0 --:--:-- --:--:-- --:--:-- 2331k
configure: WARNING: unrecognized options: --disable-rpath, --disable-rpath-hack, --disable-nls, --enable-shared, --enable-static
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for aarch64-linux-android-strip... llvm-strip
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for aarch64-linux-android-ranlib... llvm-ranlib
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether the compiler supports GNU C++... yes
checking whether aarch64-linux-android-clang++ accepts -g... yes
checking for aarch64-linux-android-clang++ option to enable C++11 features... unsupported
checking for aarch64-linux-android-clang++ option to enable C++98 features... unsupported
checking whether make supports the include directive... yes (GNU style)
checking dependency style of aarch64-linux-android-clang++... none
checking for aarch64-linux-android-gcc... aarch64-linux-android-clang
checking whether the compiler supports GNU C... yes
checking whether aarch64-linux-android-clang accepts -g... yes
checking for aarch64-linux-android-clang option to enable C11 features... none needed
checking whether aarch64-linux-android-clang understands -c and -o together... yes
checking dependency style of aarch64-linux-android-clang... none
checking for aarch64-linux-android-clang option to enable large file support... none needed
checking whether aarch64-linux-android-clang++ supports C++14 features with -std=c++14... no
checking whether aarch64-linux-android-clang++ supports C++14 features with +std=c++14... no
checking whether aarch64-linux-android-clang++ supports C++14 features with -h std=c++14... no
checking whether aarch64-linux-android-clang++ supports C++14 features with -std=c++1y... no
checking whether aarch64-linux-android-clang++ supports C++14 features with +std=c++1y... no
checking whether aarch64-linux-android-clang++ supports C++14 features with -h std=c++1y... no
configure: error: *** A compiler with support for C++14 language features is required.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-09-29 12:16:46 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/11092492729">11092492729</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/rtorrent/build.sh
+++ b/packages/rtorrent/build.sh
@@ -2,17 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://rakshasa.github.io/rtorrent/
 TERMUX_PKG_DESCRIPTION="Ncurses BitTorrent client based on libTorrent"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.9.8
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_VERSION=0.10.0
 TERMUX_PKG_SRCURL=https://github.com/rakshasa/rtorrent/releases/download/v${TERMUX_PKG_VERSION}/rtorrent-$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=9edf0304bf142215d3bc85a0771446b6a72d0ad8218efbe184b41e4c9c7542af
+TERMUX_PKG_SHA256=cc65bba7abead24151f10af116eca2342b0c320fdff3cb8d604c0af09215d3aa
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libc++, libcurl, libtorrent, libxmlrpc, ncurses"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --with-xmlrpc-c
 "
-
-# error: no member named 'bind2nd' in namespace 'std'
-termux_step_pre_configure() {
-	CXXFLAGS+=" -std=c++11"
-}

